br/pkg/lightning/errormanager/errormanager.go

```diff
@@ -122,19 +124,19 @@ const (
 
 	insertIntoConflictErrorData = `
 		INSERT INTO %s.` + ConflictErrorTableName + `
```

Comment:
When changing the table structure, need to change `ConflictErrorTableName` as well.